web/pages/group/[...slugs]/index.tsx

```diff
@@ -258,14 +242,14 @@ export default function GroupPage(props: {
       title: 'Leaderboards',
       content: leaderboardTab,
     },
-    {
-      title: 'About',
-      content: aboutTab,
-    },
     {
       title: 'Posts',
       content: postsPage,
     },
+    {
```

Comment:
I think it would be good if navigating to `/about` would show this overview tab. Helps with the ongoing tournaments for one.

I also think getting the routing right for switching between tabs is useful. E.g. clicking leaderboards should change the url to `/leaderboards` and if you hit back, it goes to the previous tab.  Probably a separate change though!